opentelemetry-otlp/Cargo.toml

```diff
@@ -68,7 +68,7 @@ serialize = ["serde"]
 default = ["grpc-tonic", "trace"]
 
 # grpc using tonic
-grpc-tonic = ["tonic", "prost", "http", "tokio", "opentelemetry-proto/gen-tonic"]
+grpc-tonic = ["tonic", "prost", "http", "tokio", "opentelemetry-proto/gen-tonic", "opentelemetry-proto/tonic-transport"]
```

Comment:
No, because `gen-tonic` is needed by `opentelemetry-otlp/http-proto` to encode the protobuf messages. `opentelemetry-otlp/http-proto` including `tonic/transport`  by default is what I'm trying to avoid here.

Reply:
Yeah, but you want to change the defaults for a relatively niche use case. I'm suggesting we shouldn't do that, and merely make it possible to address your use case, without making it the default.

Reply:
Right I see what you mean, I can add `tonic-transport` to the `opentelemetry-proto` defaults so it's a backwards-compatible change :+1:

Reply:
Yup, that sounds good!